docs/opcodes/E2.mdx

```diff
@@ -9,7 +9,7 @@ group: Stack Memory Storage and Flow Operations
 
 The program counter (PC) is a byte offset in the deployed [code](/about). It indicates which instruction will be executed next. When an [ADD](/#01) is executed, for example, the PC is incremented by 1, since the instruction is 1 byte. The [PUSH](/#60) instructions are bigger than one byte, and so will increment the counter accordingly.
 
-The **RJUMPV** instruction may alter the program counter with relative offset, thus breaking the linear path of the execution to another point in the deployed [code](/about). It is used to implement functionalities like loops and conditions. if `case > max_index` (out-of-bounds case), fall through and increment the counter by `(max_index + 1) * 2` without `relative_offset` takes into account.
+The **RJUMPV** instruction may alter the program counter with relative offset, thus breaking the linear path of the execution to another point in the deployed [code](/about). It is used to implement functionalities like switch statements. if `case > max_index` (out-of-bounds case), fall through and increment the counter by `(max_index + 1) * 2` without `relative_offset` takes into account.
 
 An interesting feature is that `RJUMPV 0 relative_offset` is an inverted-[RJUMPI](/#E1), which can be used in many cases.
```

Comment:
Hm, I think it's accurate. `RJUMPV 0 rel_offset` jumps iff `ISZERO RJUMPI rel_offset` jumps. We also have it in the EIP-4200. But I just now wrote the combinations down, so maybe missing sth.

This wording maybe is missing what EIP has: "which can be used in many cases instead of ISZERO RJUMPI relative_offset" explicitly